ote/src/cljs/ote/views/transport_operator_ytj.cljs

```diff
@@ -58,15 +58,19 @@
        :validate  [[:business-id]]
        :required? true
        :warning   (tr [:common-texts :required-field])
-       :should-update-check form/always-update}
-
-      {:name      ::t-operator/btn-submit-business-id
-       :type      :external-button
-       :label     (tr [:organization-page :fetch-from-ytj])
-       :primary   true
-       :secondary true
-       :on-click  #(e! (to/->FetchYtjOperator (::t-operator/business-id operator)))
-       :disabled  (ytj-loading? state)}
+       :should-update-check form/always-update
+       :on-blur #(e! (to/->EnsureUniqueBusinessId (-> % .-target .-value)))}
+
+      ; show ytj-button only if business-id is not taken and if business-id is valid
+      (when (and (nil? (get-in state [:transport-operator :ote.ui.form/errors ::t-operator/business-id]))
```

Comment:
Instead of hiding it might be more consistent for UI and UX to simply disable the button.

Reply:
yep. good idea. I changed this.